tests/scripts/thread-cert/border_router/MATN_04_MulticastListenerTimeout.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2021, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS 'AS IS'
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+import logging
+import unittest
+
+import pktverify
+from pktverify import packet_verifier, packet_filter, consts
+from pktverify.consts import MA1
+import config
+import thread_cert
+
+# Test description:
+# The purpose of this test case is to verify that a Primary BBR can remove an
+# entry from a Multicast Listeners Table, when the entry expires. The test case
+# also verifies that the Primary BBR accepts a new registration to the
+# previously expired multicast group.
+#
+# Topology:
+#    ----------------(eth)------------------
+#           |                  |     |
+#          BR1 (Leader) ----- BR2   HOST
+#           |                  |
+#           |                  |
+#          Router_1------------+
+#
+
+BR_1 = 1
+BR_2 = 2
+ROUTER_1 = 3
+HOST = 4
+
+MLR_TIMEOUT_MIN = 300
```

Comment:
Move this constant to config.py since it hold true for all TCs?

Reply:
added in consts.py